commands/release.go

```diff
@@ -394,14 +392,15 @@ func downloadRelease(cmd *Command, args *Args) {
 	release, err := gh.FetchRelease(project, tagName)
 	utils.Check(err)
 
-	include := args.Flag.Value("--include")
-	includeRe, err := regexp.Compile(include)
-	utils.Check(err)
-
+	pattern := args.Flag.Value("--include")
 	for _, asset := range release.Assets {
 		// if the include glob is not empty and it doesn't match the pattern
-		if include != "" && !includeRe.MatchString(asset.Name) {
-			continue
+		if pattern != "" {
+			isMatch, err := filepath.Match(pattern, asset.Name)
+			utils.Check(err)
+			if !isMatch {
+				continue
```

Comment:
Should the `hub release download` process exit with an error if no assets matched the pattern? The stderr output could potentially print the available files

Reply:
sounds good, I was going to add a message if there were no assets at all but I don't think that's possible since a release will contain the source code at the time.